example/tpm_ibmtss2/README.md

```diff
@@ -0,0 +1,92 @@
+Test script for TPM 2
+
+Usage
+-----
+
+The TPM 2 test script loads the RSA key into the TPM's NULL hierarchy. Please
+make sure that the NULL hierarchy has no password set and no other
+application is using the TPM 2 and sufficient space is available in the TPM for
+loading the RSA key. Since access to TPM devices is typically restricted to the
+root user, the test script (time_decrypt) will need to be run as root.
+
+The TPM 2 test script can for example be run against swtpm using the vtpm
+proxy device:
+
+ > modprobe tpm_vtpm_proxy
+ > swtpm chardev --vtpm-proxy --tpmstate dir=./ --tpm2
+ New TPM device: /dev/tpm1 (major/minor = 253/1)
+
+For the test script to use the proper device, which would be /dev/tpmrm1,
+set the following environment variable:
+
+ TPM_DEVICE=/dev/tpmrm1
+
+
+When using a hardware TPM 2 the following should be used:
+
+ TPM_DEVICE=/dev/tpmrm0
+
+The following packages are needed:
+
+ - openssl-devel / libssl-dev
+ - tss2-devel / libtss2-dev
+ - swtpm-tools
+
+Run `step0.sh`, `step1.sh` as normal. Instead of running `step2.sh` run
+the `step2-marvin.sh` script.
```

Comment:
why `step2-marvin.sh`? does swtpm implement implicit rejection?

Reply:
I guess it depends on what version of OpenSSL is on the system, no?

Reply:
correct